rpcd/playbooks/roles/rpc_maas/templates/elasticsearch_process_check.yaml.j2

```diff
@@ -0,0 +1,19 @@
+type: agent.plugin
```

Comment:
Since it's thrown more than a few of us, someone file a bug to clean them up. We can lump it into a tech debt swimlane in the futures.